src/engagement_db_to_analysis/data_wrangling_functions.py

```diff
@@ -0,0 +1,81 @@
+from core_data_modules.util import TimeUtils
+
+from core_data_modules.data_models.code_scheme import CodeTypes
+from core_data_modules.cleaners.cleaning_utils import CleaningUtils
+from core_data_modules.traced_data import Metadata
+from core_data_modules.logging import Logger
+
+
+log = Logger(__name__)
+
+def _impute_age_category(user, messages_traced_data, analysis_dataset_configs):
+    """
+    Imputes age category for age dataset messages.
+
+    :param user: Identifier of user running the pipeline.
+    :type user: str
+    :param messages_traced_data: Messages TracedData objects to impute age_category.
+    :type messages_traced_data: list of TracedData
+    :param analysis_dataset_config: Analysis dataset configuration in pipeline configuration module.
+    :type analysis_dataset_config: pipeline_config.analysis_configs.dataset_configurations
+    """
+
+    log.info("Imputing age categories for age dataset messages...")
```

Comment:
I think we should say what the dataset actually is, as it might not exactly be called 'age'

Reply:
updated thanks